docs/internal/investigations/runtime-governor-poc/README.md

```diff
@@ -0,0 +1,138 @@
+# Runtime Governor concept with Dapr
+
+This document describes the scenario that will be developed to prove that [Dapr](https://dapr.io/) sidecars can be configured from a central place.
+
+## Reasons
+
+Kyma Runtimes must consume fewer resources. It can be achieved by delegating some of Kyma's responsibilities to Dapr. 
+Dapr injects sidecars into selected Pods where they fulfill their given tasks, such as state management or pub-sub.
+Configuration for these sidecars can be held in one central place from which Kyma Runtimes could fetch them and configure the sidecars accordingly.
+
+## PoC scenario
+
+The scenario will present two Runtimes using [Dapr State API](https://github.com/dapr/docs/blob/master/reference/api/state_api.md) to persist the state in two different Redis instances.
```

Comment:
I feel like I'm missing a concrete description of the list below. Maybe we could add something like this?
```suggestion
The scenario will present two Runtimes using [Dapr State API](https://github.com/dapr/docs/blob/master/reference/api/state_api.md) to persist the state in two different Redis instances. It consist of the following steps:
```
If this does not work, could you propose something more appropriate?